meerk40t/gui/scene/scenepanel.py

```diff
@@ -141,30 +112,24 @@ def on_mousewheel(self, event):
         """
         ScenePanel mousewheel event.
 
-        If modifiers are present it calls the event 'wheelup_ctrl' or 'wheeldown_ctrl' this also triggers scene events
-        for up, down, and left and right which exist on some mice and trackpads. If shift is held down while the wheel
-        event occurs the up and down rotation is treated as left and right.
+        Triggers scene events for up, down, and left and right which exist on some mice and trackpads. If shift is held
+        down while the wheel event occurs the up and down rotation is treated as left and right.
 
         """
         if self.scene_panel.HasCapture():
             return
         rotation = event.GetWheelRotation()
+        modifiers = self.event_modifiers(event)
         if event.GetWheelAxis() == wx.MOUSE_WHEEL_VERTICAL and not event.ShiftDown():
```

Comment:
Not sure while you are not using ```"shift" in modifiers``` - would seem more consistent to me. But maybe just a matter of style

Reply:
Sort of. The axis there is more fundamental than the modifiers. If you hold shift your axis is inverted, that's not really part of the bag of modifiers. Also, I think event.ShiftDown() is just a check of a bool rather than `"shift" in modifiers` which I guess is a hash-table lookup and also constant time (actually a list, I guess it's n-time). But, here we're dealing with the panel itself and the fundamentals of how the panel work, and the modifiers for passing to the scene.

I had originally considered it, but I figured this was easier to read. And even if modifiers got changed or whatever, this isn't part of that code that governs how scenes interact but rather part of the scenepanel which is tasked with taking the interactions and handing them to the scene to process. `Modifiers` is part of the scene processing and really part of the hook into those bindings.